src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -7347,4 +7430,22 @@ definitions:
           - DONE
           - MAX_CONTENT_SIZE_EXCEEDED
         default: PENDING
-        description: the current status of the download node creation
\ No newline at end of file
+        description: the current status of the download node creation
+  ClientBody:
+    type: object
+    properties:
+      client:
+        type: string
+        description: the client name
+  PasswordResetBody:
+    type: object
```

Comment:
missing required declaration? I would have thought all properties in the body were mandatory.